demo/admin/src/products/future/ProductForm.cometGen.ts

```diff
@@ -35,12 +35,22 @@ export const ProductForm: FormConfig<GQLProduct> = {
             collapsible: true,
             initiallyExpanded: false,
             fields: [
+                {
+                    type: "asyncSelect",
+                    virtual: true,
+                    name: "manufacturerCountry",
+                    gqlFieldName: "manufacturer",
+                    initQueryIdPath: "addressAsEmbeddable.country",
+                    initQueryLabelPath: "addressAsEmbeddable.country",
```

Comment:
do you mean how it has to be used? we could provide support via typescript with the dot-nested object-structure strings we already had in another pull-request.

Reply:
@nsams @johnnyomair we still need to simplify how this should be configured. I think there are two main aspects: the initial-query needs to be customizable, and the returning structure needs to be known in admin-generator to correctly access the queried id and label.

allowing to directly provide the graphql selection/query would be easy to understand but hard to detect how the structure has to be used for admin-generator to correctly generate the data-access.

Reply:
Do you have alternative suggestions how this could be done?

Reply:
No, still not. Maybe we should for now accept this solution, because I can't wait until we have a better idea. And I'm afraid we wont find a better solution in near future...